src/loader.ts

```diff
@@ -9,34 +9,46 @@ const headers = (apiKey: string) => ({
 
 const apiHash = (apiKey: string) => base64Encode(apiKey);
 
+export const DEFAULT_TIMEOUT = 10000;
```

Comment:
@jdwyah thoughts on the right (ms) value here?

Reply:
🎱  Thinking out loud. I would like 99th percentile to be 200ms or so with good internet.  Fastly probably 100ms. We're really talking 2 scenarios.  1) They have bad internet.  10 sec seems reasonable.  2) they have good internet and there is an issue. 10 sec timeout for fastly, then another 10 for us.  We could speed that failover up with a 5 sec timeout or even more aggressive. But now we are maybe hurting people with flaky terrible connections.    tl;dr I think 10 is reasonable. I can see some people configuring it much lower if they are comfortable with fallbacks. But this seems safer.

Reply:
💯 my thinking as well. This should never be hit, but it'll be a good safety mechanism.